src/languageProvider/locales/en_US.json

```diff
@@ -120,9 +113,8 @@
   "str.m": "m",
   "str.minutes": "Minutes",
   "str.name": "Name",
-  "str.notEnoughBot": "You don't have enough BOT",
-  "str.notEnoughQtum": "You don't have enough QTUM",
-  "str.minutes": "Minutes",
+  "str.notEnoughBot": "Not enough BOT",
```

Comment:
I feel like Not enough BOT sounds a bit cold

Reply:
Didn't change this either. I think something is up with the script?